CHANGELOG.md

```diff
@@ -48,7 +49,7 @@ Re-wrote whole pipeline in nextflow [DSL2](https://www.nextflow.io/docs/latest/d
 * [#229](https://github.com/nf-core/mag/pull/229) - `--classifier_removeHash` and `--qiime_timezone` became unnecessary
 * [#229](https://github.com/nf-core/mag/pull/229) - `--onlyDenoising` is deprecated in favour of `--skip_taxonomy` (which does the exact same thing)
 * `--taxon_reference` became unnecessary
-* `--reference_database`, `--dereplication` are currently not supported as they are QIIME2 specific. Either a more general solution will be supported soon or QIIME2 classification will be removed.
+* [#229](https://github.com/nf-core/mag/pull/229) - `--reference_database` and `--dereplication` are currently not supported any more. `--qiime_ref_taxonomy` allows now choosing a taxonomic reference
```

Comment:
I suggest you remove either "currently" or "any more".

Reply:
Thanks, I changed it!